parameter_tools/src/main/java/coppercore/parameter_tools/JSONSync.java

```diff
@@ -26,11 +33,31 @@ private FileReader getFileReader(String path) {
         }
     }
 
+    private FileWriter getFileWriter(String path) {
+        try {
+            return new FileWriter(path);
+        } catch (IOException e) {
+            throw new RuntimeException("IOException " + path, e);
+        }
+    }
+
     @SuppressWarnings("unchecked")
     public void loadData() {
         instance = gson.fromJson(getFileReader(file), (Class<T>) instance.getClass());
     }
 
+    public void saveData() {
+        String json = gson.toJson(instance);
```

Comment:
I will double check

Reply:
I will double check and also it will allow use to quickly generate the json without having to write the whole json and if we serialized the wpilib objects it will generate the structure for us

Reply:
@linglejack06 it currently does not serialize with exclude, but i will fix that soon

Reply:
json exclude works confirmed by nathan